docs/parameters.rst

```diff
@@ -72,3 +72,7 @@ Here is the full list of configuration parameters you can specify in a ``config.
 
 **over_zoom**: int
  	An integer greater than 0. If set for XYZ tiles, it will fetch tiles from `zoom` + `over_zoom`, to create higher resolution tiles which fill out the bounds of the original zoom level.
+
+**band_indicies**: list
+	A list of band incidies to pull from a tiff. Defaults to (1, 2, 3) if not specified. For using non-RGB tifs, ie MS tifs this should be over-written
```

Comment:
What happens if this is specified for non-tifs? We should also define MS + spacenet